### PR TITLE
Autoplay videos and GIFs if enabled by the user.

### PR DESCRIPTION
**Pull request by Half-Shot**
(Also mutes the video, because nobody wants that).

Fixes https://github.com/vector-im/riot-web/issues/2433

### CODE REVIEW

**Comment by dbkr** [comment]
@matrixbot test this please

**Comment by Half-Shot** [comment]
Signed-off-by: Will Hunt <half-shot@molrams.com>

File: src/components/views/messages/MImageBody.js [hunk 1]
```diff
@@ -100,7 +104,9 @@ module.exports = React.createClass({
 
     _getThumbUrl: function() {
         const content = this.props.mxEvent.getContent();
+        console.log(content);
```
Comment (dbkr) [inline src/components/views/messages/MImageBody.js]: stray log line

File: src/components/views/messages/MImageBody.js [hunk 2]
```diff
@@ -70,22 +71,25 @@ module.exports = React.createClass({
 
     _isGif: function() {
         const content = this.props.mxEvent.getContent();
-        return (content && content.info && content.info.mimetype === "image/gif");
+        return (
+          content &&
+          content.info &&
+          content.info.mimetype === "image/gif");
```
Comment (dbkr) [inline src/components/views/messages/MImageBody.js]: nitpick: bracket generally goes on a new line

File: src/components/views/messages/MImageBody.js [hunk 3]
```diff
@@ -100,7 +104,9 @@ module.exports = React.createClass({
 
     _getThumbUrl: function() {
         const content = this.props.mxEvent.getContent();
+        console.log(content);
         if (content.file !== undefined) {
+            // Don't use the thumbnail for client's wishing to autoplay gifs.
```
Comment (dbkr) [inline src/components/views/messages/MImageBody.js]: image

**Pull request merged**